moveit_setup_assistant/test/moveit_config_data_test.cpp

```diff
@@ -77,49 +77,52 @@ class MoveItConfigData : public testing::Test
   moveit::core::RobotModelPtr robot_model;
 };
 
-TEST_F(MoveItConfigData, ReadingControllers)
-{
-  boost::filesystem::path res_path(MOVEIT_TEST_RESOURCES_DIR);
+// TODO: after removing the robot namespace from ros_controllers.yaml, inputROSControllersYAML needs to be refactored
+// accordingly
```

Comment:
disabling tests is bad practice. can you just refactor inputROSControllersYAML now?

Reply:
I tried tho but it wasn't so trivial, so as to get this PR merged as fast as possible I thought of postponed it to a later PR.
I can try to allocate time to do it across this weekend  :thinking:

Reply:
Done